js/src/configuration-wizard.js

```diff
@@ -115,21 +116,37 @@ class App extends React.Component {
 	}
 
 	/**
-	 * Renders the App componetn.
+	 * Renders the App component.
 	 *
 	 * @returns {JSX.Element|null} The rendered app component.
 	 */
 	render() {
+		// When the wizard is loading, don't do anything.
+		if ( this.state.isLoading === true ) {
+			return null;
+		}
 
-		if ( this.state.isLoading === false && this.state.config !== {} ) {
-
+		// When there is a config and it's not empty.
+		if ( typeof ( this.state.config ) !== "undefined" && this.state.config !== {} ) {
 			return (
 				<div>
 					<OnboardingWizard { ...this.state.config }/>
 				</div>
 			);
 		}
-		return null;
+
+		let message = {
```

Comment:
Should there be a translation note for translators about the `{{link}}` addition in the code?